### ISSUE TITLE
optimization by --linemerge

### ISSUE THREAD

**Issue by ithinkido** [abey79/vpype#34]
When using --linemerge to connect multiple line segments to one single line, the output still contains the old line nodes, and the result is one line, but with multiple nodes. when plotting a line like this the plotter slows down and speed up for each node and the result is a kind of 'ghost' line in the plot. Is it possible to output a mergerd line as a single line between a start and end node, creating the longest possibe single straight line?
![image](https://user-images.githubusercontent.com/52834821/81501868-df979300-92da-11ea-9622-999625facaf7.png)
![image](https://user-images.githubusercontent.com/52834821/81501880-f1793600-92da-11ea-9cb1-b2df553859b0.png)
[linetest.zip](https://github.com/abey79/vpype/files/4605793/linetest.zip)

**Comment by abey79** [comment]
`linemerge` doesn't attempt to simplify lines. When two lines' endings are close together, they are concatenated and their endings replaced by the mean point, so two collinear 1-segment lines are converted to a single, 3-point line. You want to use `linesimplify` after `linemerge` to remove those spurious points.

(Note: the Axidraw isn't at all affected by spurious points -- I'm surprised it's the case for other plotters.)

**Comment by ithinkido** [comment]
Have I understood this correctly,  linemerge will produce a 3- point line (start , mean, end), and linesimplify will remove the mean point and produce a 2-point line ( start and end)?
Just tried with:
 vpype read 1.svg linemerge --tolerance 0.1mm linesimplify linesort write --layer-label Pen%d --page-format 191x272mm 1_opt.svg
and I still have what looks like the mean point. Is there a way to output only start and end point lines?

**Comment by abey79** [comment]
On second look, I'm seeing issues with `linemerge` on your file, apparently an issue in Shapely. I've pushed a quick fix that seems to address the situation.

This is what happens now:

`vpype read 1.svg  linemerge  show -dac -u mm`

<img width="752" alt="image" src="https://user-images.githubusercontent.com/49431240/81502601-399a5780-92df-11ea-8627-42d9b1316bba.png">

`vpype read 1.svg  linemerge linesimplify  show -dac -u mm`

<img width="752" alt="image" src="https://user-images.githubusercontent.com/49431240/81502615-43bc5600-92df-11ea-8a3f-d3cd26e309ec.png">


Side note: while looking into that, I got confused with what happens inside the windows, until I realised that there are overlapping lines there. Is that on purpose?
